src/apps/bridge/sensor_drivers/aanderaaSensor.cpp

```diff
@@ -53,7 +53,7 @@ void AanderaaSensor::aanderaSubCallback(uint64_t node_id, const char *topic, uin
         uint64_t sensor_reading_time_sec = d.header.sensor_reading_time_ms / 1000U;
         uint32_t sensor_reading_time_millis = d.header.sensor_reading_time_ms % 1000U;
 
-        int8_t node_position = topology_sampler_get_node_position(node_id, 1000);
+        int8_t node_position = topology_sampler_get_node_position(node_id, pdTICKS_TO_MS(5000));
```

Comment:
I changed this to 5 seconds since now we will try to get the node position and take the mutex infrequently and thus we want to make sure we actually get it and don't log an invalid value (unless we know for sure it is not in our topology).